library/dnet.c

```diff
@@ -46,6 +46,29 @@
 
 #include "logger.hpp"
 
+int dnet_remove_local_new(struct dnet_backend_io *backend, struct dnet_node *n, struct dnet_id *id,
+			  void *packet, size_t packet_size)
+{
+	const size_t cmd_size = sizeof(struct dnet_cmd) + packet_size;
+	int err;
+	char buffer[cmd_size];
+	struct dnet_cmd *cmd = (struct dnet_cmd *)buffer;
```

Comment:
Obviously, it is used to allocate space for cmd that have type `struct dnet_cmd`.
I don't want to use alloca().